s3backgrounddelete/s3backgroundconsumer.service

```diff
@@ -25,6 +25,7 @@ Environment="PYTHONPATH=/usr/lib/python3.6/site-packages/"
 Type=simple
 ExecStart=/opt/seagate/cortx/s3/s3backgrounddelete/s3backgroundconsumer
 ExecStop=/bin/kill -TERM $MAINPID
+ExecReload=/bin/kill -USR1 $MAINPID
```

Comment:
Similar changes required in file ".../s3backgrounddelete/s3backgroundproducer.service".

Reply:
Done in the following commit 7d5def8322ba3878f2a26cd425a53a39677446a2